trinity/sync/common/chain.py

```diff
@@ -134,6 +166,16 @@ def _get_peer_header_syncer(self, peer: BaseChainPeer) -> Iterator['PeerHeaderSy
                 new_headers = tuple(h for h in header_batch if h not in self.header_queue)
                 await self.wait(self.header_queue.add(new_headers))
 
+    def syncing(self) -> Union[bool, SyncProgress]:
+        if not self._syncing:
+            return False
+        return self._peer_header_syncer.sync_progress
+
+    async def handle_syncing_requests(self) -> None:
```

Comment:
Thoughts on renaming this to `handle_sync_status_requests` or `handle_sync_info_requests`.  The current name could easily be confused as something which is related to the actually code which performs chain sync.